secant/UIComponents/CircularFrame/CircularFrame.swift

```diff
@@ -8,22 +8,44 @@
 import SwiftUI
 
 struct CircularFrame: View {
+    @Environment(\.colorScheme) var colorScheme
+
     var body: some View {
         GeometryReader { proxy in
-            let lineWidth = proxy.size.width * 0.05
-
+            let lineWidth = proxy.size.width * 0.06
+            
```

Comment:
[non-blcoking | nitpick | typo] whitespace (newline is good! just the spaces to delete not sure if the suggestion below actually does that properly 😅)

```suggestion

```

Reply:
I removed it on my side